bibangamba/GitHubUsers
# #156004609 Pass Data From MainActivity to DetailActivity

#### What does this PR do?
Pass Data From MainActivity to DetailActivity

#### Description of tasks to be done?
- create GithubUserDetailsPresenter class and GithubDetailsView interface to handle user details
- add a new endpoint in the GithubUsersAPI class to retrieve a specific user's data
- populate the details layout with retrieved user data

#### How can this be manually tested?
- clone the branch and run the application. It should start out on the list of GitHub users.
- click on any user in the list, it should show their information on the details page

What are the relevant pivotal tracker stories?
[#156004609](https://www.pivotaltracker.com/story/show/156004609)

#### Screenshots
![image](http://g.recordit.co/ds4QfCJ9Ws.gif)
